index.html

```diff
@@ -6,167 +6,25 @@
   <title>Bynder media selector</title>
   <!-- Kontent Styling -->
   <link rel="stylesheet" type="text/css"
-    href="https://kentico.github.io/kontent-custom-element-samples/shared/custom-element.css">
+    href="https://kentico.github.io/kontent-custom-element-samples/shared/custom-element-v2.css">
```

Comment:
do we want to rely on the deployed version?

Reply:
yes, we want!